Client/src/Utils/stringUtils.js

```diff
@@ -0,0 +1,3 @@
+export const capitalizeFirstLetter = (str) =>{
+  return  str.charAt(0).toUpperCase() + str.slice(1);
+}
```

Comment:
I suggest adding some safetry checks to this utility function.  If we've gone through the trouble to extract it we should make it safe to use.

On visual inspection It will throw errors right now if given the wrong type, empty string, `null`, or `undefined`.  Could be other edge cases too, I would explore and make this robust.

Reply:
will do and some JSdoc as well

Reply:
resolved in : https://github.com/bluewave-labs/bluewave-uptime/pull/1149/commits/a9aece6123eaa20ce2173c62c7f7904379a9f0dd
https://github.com/bluewave-labs/bluewave-uptime/pull/1149/commits/4e60981b0b1c487182b830991c89c117f6670006 etc

Reply:
yes, will update